lib/main.js

```diff
@@ -36,6 +37,7 @@ const options = {
   downloadApi: process.env.RESINOS_TESTS_RESINIO_STAGING_API_URL,
   delta: process.env.RESINOS_TESTS_RESIN_SUPERVISOR_DELTA,
   sshKeyLabel: process.env.RESINOS_TESTS_SSH_KEY_LABEL,
+  resultsFilePath: process.env.RESINOS_TESTS_RESULTS_PATH,
```

Comment:
Do we have to set this to something in the CI services? Can we modify the script to still publish even if the user is not interested in the results?

Reply:
This variable has a constant value define in `entry.sh`. It is not configurable at this point.